robot/Cumulus/tests/browser/gift_entry/std_object_related_lookups_fields_bge.robot

```diff
@@ -0,0 +1,165 @@
+	...						 and verify that other related fields automatically load on the gift entry form. Remove default from batch and verify
+	...						 leads fields are not autopopulated. Create a gift using new lead in lookup and verify correct related values load
+	...						 automatically and on save batch table has correct lead values. create another gift with different lead and verify
+	...						 lead values load and save correctly. Open previous gift from table and verify correct lead info loads on form
+	[tags]                                        unstable      feature:GE        W-043224
+	# Create object and field mappings in Advanced Mapping
+	Click Configure Advanced Mapping
+	Create New Object Group                       Lead
+	...                                           objectName=Lead (Lead)
+	...                                           relationshipToPredecessor=Child
+	...                                           ofThisMappingGroup=Account 1
+	...                                           throughThisField=Account Lookup (Account_Lookup__c)
+	...                                           importedRecordFieldName=Lead Lookup (Lead_Lookup__c)
+	...                                           importedRecordStatusFieldName=Lead Imported Status (Lead_Imported_Status__c)
+	View Field Mappings Of The Object             Lead
+	Create Mapping If Doesnt Exist                Lead Company (Lead_Company__c)  	  	Company (Company)
+	Create Mapping If Doesnt Exist                Lead Last Name (Lead_Last_Name__c)    Last Name (LastName)
+	Reload Page
+	#Create a template with newly mapped lead lookup, company and lastname fields and add fields to batch table
+	Go To Page                                    Landing                GE_Gift_Entry
+	Click Link                                    Templates
+	Click Gift Entry Button                       Create Template
+	Current Page Should Be                        Template               GE_Gift_Entry
+	Enter Value In Field
+	...                                           Template Name=${TEMPLATE_NAME}
+	...                                           Description=This template is created by an automation script.
+	Click Gift Entry Button                       Next: Form Fields
+	Perform Action on Object Field                select  				 Lead           Lead Lookup
+	Perform Action on Object Field                select  				 Lead           Company
+	Perform Action on Object Field                select  				 Lead           Last Name
+	Click Gift Entry Button						  Next: Batch Settings
+	Add Batch Table Columns          		      Lead: Company     	 Lead: Last Name       Data Import: Lead Lookup
+	Click Gift Entry Button          			  Save & Close
+	Current Page Should Be           			  Landing                GE_Gift_Entry
+	Click Link                                	  Templates
+    Wait Until Page Contains                  	  ${TEMPLATE_NAME}
+    Store Template Record Id                  	  ${TEMPLATE_NAME}
```

Comment:
Indentation seems to be off in a few places in this file.. Can you fix it ?

Reply:
looks fine in my ide so not sure why it shows like this here, any idea?

Reply:
yeah indentation seems little off to me too. But don't know why it happens.

Reply:
Hmm.. I don't know. Could a difference between "spaces" and "tabs"..

Reply:
Out of curiosity, do we have an internal robot files style guide?